samples/sampler/.qa-storybook/preview.js

```diff
@@ -0,0 +1,22 @@
+import {addDecorator} from '@storybook/react';
+import {configureActions} from '@enact/storybook-utils/addons/actions';
+import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
+import {themes} from '@storybook/theming';
+import {withKnobs} from '@enact/storybook-utils/addons/knobs';
+
+import ThemeEnvironment from '../src/ThemeEnvironment'
+
+configureActions();
+addDecorator(withKnobs);
+export const parameters = {
+	knobs: {
+		timestamps: true
+	},
+	docs: {
+		container: DocsContainer,
+		page: DocsPage,
+		iframeHeight: 300,
+		theme: themes.light
+	}
```

Comment:
```suggestion
	},
	options: {
		storySort: {
			method: 'alphabetical'
		}
	}
```

You can refer to the latest changes for the sandstone storybook. (https://github.com/enyojs/sandstone/pull/832/files)